easybuild/easyconfigs/n/nanopolish/nanopolish-0.10.2-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,46 @@
+# easybuild easyconfig
+#
+# John Dey jfdey@fredhutch.org
+# Fred Hutchenson Cancer Research Center
+#
+#  June 13, 2017 0.7.1
+#  Dec 31, 2018, 0.10.2
+easyblock = 'MakeCp'
+
+name = 'nanopolish'
+version = '0.10.2'
+versionsuffix = '-Python-2.7.15'
+
+homepage = 'http://github.com/jts/nanopolish'
+description = """Software package for signal-level analysis of Oxford Nanopore
+ sequencing data."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = [('https://github.com/jts/nanopolish/archive/')]
+sources = [
+    {'download_filename': 'v%(version)s.tar.gz',
+     'filename': SOURCE_TAR_GZ}
+]
+checksums = ['c0ba8750df379af51a7f3108ea990096c2684e8c20b75bf76f0ed278a84f889a']
+
+buildopts = 'HDF5=local EIGEN=local HTS=local'
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('HDF5', '1.10.2'),
+    ('Eigen', '3.3.5', '', True),
+    ('HTSlib', '1.9'),
+    ('fast5', '0.6.5', '', True),
+]
+
+files_to_copy = [(['nanopolish'], 'bin'), 'scripts']
+
+
```

Comment:
@fizwit Please clean up these duplicate empty lines (same below), just one empty line is enough

Reply:
I am removing the extra blank lines